config/formats.js

```diff
@@ -280,9 +280,9 @@ let Formats = [
 			'Alcremie-Gmax', 'Appletun-Gmax', 'Charizard-Gmax + Solar Power', 'Coalossal-Gmax', 'Copperajah-Gmax',
 			'Corsola-Base', 'Darmanitan-Base', 'Darumaka-Base', 'Diglett-Alola', 'Dugtrio-Alola', 'Duraludon-Gmax',
 			'Farfetch\'d-Base', 'Flapple-Gmax', 'Garbodor-Gmax', 'Gengar-Gmax', 'Grimmsnarl-Gmax', 'Hatterene-Gmax',
-			'Kingler-Gmax', 'Lapras-Gmax', 'Linoone-Base', 'Machamp-Gmax', 'Ninetales-Alola', 'Orbeetle-Gmax',
-			'Ponyta-Base', 'Rapidash-Base', 'Slowpoke-Galar', 'Stunfisk-Base', 'Toxtricity-Gmax', 'Vulpix-Alola',
-			'Zigzagoon-Base',
+			'Kingler-Gmax', 'Lapras-Gmax', 'Linoone-Base', 'Machamp-Gmax', 'Meowth-Alola', 'Ninetales-Alola',
+			'Orbeetle-Gmax', 'Persian-Alola', 'Ponyta-Base', 'Rapidash-Base', 'Slowpoke-Galar', 'Stunfisk-Base',
+			'Toxtricity-Gmax', 'Vulpix-Alola', 'Zigzagoon-Base',
```

Comment:
Shouldn't Toxtricity-Low-Key-Gmax be banned too

Reply:
It actually is banned right now, but for the wrong reason (it says it's because it's not part of Galar dex).